io/eolearn/io/processing_api.py

```diff
@@ -190,7 +191,7 @@ class SentinelHubInputTask(SentinelHubInputBase):
 
     def __init__(self, data_source, size=None, resolution=None, bands_feature=None, bands=None, additional_data=None,
                  maxcc=1.0, time_difference=None, cache_folder=None, max_threads=None, config=None,
-                 bands_dtype=np.float32, single_scene=False, mosaicking_order='mostRecent'):
+                 bands_dtype=np.float32, single_scene=False, mosaicking_order='mostRecent', aux_input_args=None):
```

Comment:
I just wonder if name `aux_input_args` is clear enough. What about `aux_data_source_args`? It's even more verbose but it tells you parameters are connected with a data source.

Feel free to disregard this comment if you don't agree.

Reply:
I'm not sure the `data_source` part makes it any clearer. Perhaps `aux_sh_request_args` in this case? In any case, hopefully users check the docstring if anything is unclear. I'm not sure you can explain the usage of this parameter with the name itself anyway.

Reply:
A name `aux_request_args` seems clear for me. (`sh` in the name is sort of implicit since you are using `SentinelHubInputTask`).